doc/sphinx/source/api/CeedBasis.rst

```diff
@@ -1,5 +1,5 @@
-CeedBasis
---------------------------------------
+CeedBasis: discrete element bases and quadrature
```

Comment:
@jedbrown  are these subtitles meant to be added also to the `backend/` and `internal/` pages or not?

Reply:
I don't think that duplication is necessary; anyone looking at those pages would know what the objects are.